src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/Details.cshtml

```diff
@@ -57,71 +58,76 @@
+            </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+            <tr class="govuk-table__row">
+                <td class="govuk-table__cell">Barry Block</td>
+                <td class="govuk-table__cell">15 Apr 2002</td>
+                <td class="govuk-table__cell">Oct 2019 to Nov 2020</td>
+                <td class="govuk-table__cell govuk-table__cell--numeric">&pound;10,000</td>
+                <td class="govuk-table__cell govuk-table__cell--numeric">
+                    <a href="">Edit <span class="govuk-visually-hidden">Barry Block's details</span></a>
+                    <a href="">Delete <span class="govuk-visually-hidden">Barry Block's details</span></a>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+
+    <h3 class="govuk-heading-m">3 x Bespoke Saddler, Level 3 (Standard)</h3>
+
+    <table class="govuk-table">
+        <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+                <th scope="col" class="govuk-table__header das-table-cell-width-25">Name</th>
+                <th scope="col" class="govuk-table__header das-table-cell-width-25">Date of birth</th>
+                <th scope="col" class="govuk-table__header das-table-cell-width-30">Training Dates</th>
+                <th scope="col" class="govuk-table__header govuk-table__header--numeric das-table-cell-width-10">Price</th>
+                <th scope="col" class="govuk-table__header das-table-cell-width-10"><span class="govuk-visually-hidden">Action</span></th>
+            </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+            <tr class="govuk-table__row">
+                <td class="govuk-table__cell">Barry Block</td>
+                <td class="govuk-table__cell">15 Apr 2002</td>
+                <td class="govuk-table__cell">Oct 2019 to Nov 2020</td>
+                <td class="govuk-table__cell govuk-table__cell--numeric">&pound;10,000</td>
+                <td class="govuk-table__cell govuk-table__cell--numeric">
+                    <a href="">Edit <span class="govuk-visually-hidden">Barry Block's details</span></a>
+                    <a href="">Delete <span class="govuk-visually-hidden">Barry Block's details</span></a>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+
+    <table class="govuk-table">
+        <tbody class="govuk-table__body">
+            <tr class="govuk-table__row">
+                <th class="govuk-table__header das-table-cell-width-80" scope="row">Total apprenticeship training price <br>(excluding VAT)</th>
+                <td class="govuk-table__cell govuk-table__header--numeric das-table-cell-width-10"><strong>&pound;42,000</strong></td>
+                <td class="govuk-table__cell das-table-cell-width-10" aria-hidden="true"></td>
+            </tr>
+        </tbody>
+    </table>*@
+
+@if (Model.CanAmendCohort)
```

Comment:
@Pauljgraham what is the reason for this condition? In the current epic, we are building an editable version of this page, so I don't think we need it. Also, the logic seems to use the old EditStatus field. We should not be exposing this field through our v2 api.